.github/workflows/slash_commands.yml

```diff
@@ -218,6 +236,7 @@ jobs:
             @${{ github.event.comment.user.login }}, available slash commands include:
 
             - `/stdlib check-files` - Check for required files.
+            - `/stdlib commands-comment` - Print `make` commands for package changed in PR.
```

Comment:
@Planeshifter Can we rename this to `make-commands`? The `-comment` suffix seems to leak an implementation detail.

Reply:
Done.